### PR TITLE
refactor(stepper): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

File: src/components/calcite-stepper/calcite-stepper.scss [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-  flex-direction: row;
-  flex-wrap: wrap;
-  position: relative;
-  justify-content: space-between;
-  align-items: stretch;
-  width: 100%;
-  min-width: 100%;
+  @apply flex 
+    flex-row 
+    flex-wrap
```
Comment (jrowlingson) [inline src/components/calcite-stepper/calcite-stepper.scss:4]: Does this need to be set?

File: src/components/calcite-stepper/calcite-stepper.scss [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+    justify-between 
+    items-stretch 
+    w-full 
+    min-w-full;
 }
 
 :host([layout="vertical"]) {
-  flex-direction: column;
+  @apply flex-col;
   flex: 1 auto auto;
```
Comment (jrowlingson) [inline src/components/calcite-stepper/calcite-stepper.scss:14]: `flex-auto` ?

File: src/components/calcite-stepper/calcite-stepper.scss (hunk 1 above)
Comment (julio8a) [reply src/components/calcite-stepper/calcite-stepper.scss:4]: I am not sure, just copied over what was in the styles. You don't think we need them?

File: src/components/calcite-stepper/calcite-stepper.scss (hunk 2 above)
Comment (julio8a) [reply src/components/calcite-stepper/calcite-stepper.scss:14]: done

File: src/components/calcite-stepper/calcite-stepper.scss (hunk 1 above)
Comment (jrowlingson) [reply src/components/calcite-stepper/calcite-stepper.scss:4]: probably not - those are the defaults

Comment (jrowlingson) [reply src/components/calcite-stepper/calcite-stepper.scss:4]: we may not need flex-row - it is the initial value

**Review by jrowlingson — APPROVED** [review]

**Pull request merged**